src/earthengine/ee_worker_utils.js

```diff
@@ -116,3 +124,49 @@ export const getClassifiedImage = (eeImage, { legend = [], params }) => {
 
     return { eeImage: zones, params: { min, max, palette } }
 }
+
+// Apply filter to image collection
+export const applyFilter = (collection, filter = []) => {
```

Comment:
Seems like it should be named `applyFilters` and the 2nd arg should be named `filters` ?